load_file.txt

```diff
@@ -0,0 +1,12 @@
+------------------------------
```

Comment:
Its better to include a screen shot of sample output rather having it as part of your files.